.github/workflows/follow-nixpkgs.yml

```diff
@@ -0,0 +1,43 @@
+name: "Update flake.lock"
+on:
+  schedule:
+    - cron: "0 22 * * 6"
+  workflow_dispatch:
+
+jobs:
+  pre-job:
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@v5
+        with:
+          # skip cron
+          do_not_skip: '["workflow_dispatch"]'
```

Comment:
I think this will create duplicate pr's if there were commits after creating the first pr but the pr remains open. But whatever